### DISCUSSION TITLE
Allow ldap.toml file using relative path in config.ini

### DISCUSSION THREAD

**Discussion by changhuixu** [grafana/grafana#36979] [General]
* **I'm submitting a ...**
- [x] Feature request/ or improvement

Please include this information:
- What Grafana version are you using?
3.1.1
- What datasource are you using?
Promethus
- What OS are you running grafana on?
Windows
- What was the expected result?
Currently, we have to put a full path for ldap.toml .
Could you modify "/pkg/setting/setting.go" 
```
LdapConfigFile = ldapSec.Key("config_file").String()
```
to be
```
LdapConfigFile = makeAbsolute(ldapSec.Key("config_file").String(), HomePath)
```

Thanks a lot.

**Comment by daniellee** [comment]
Think this was fixed a long time ago. Closing.

**Comment by daniellee** [comment]
Maybe I was too fast - is this different on Windows? A relative path works on Linux.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
